frontend_app_kanban/src/component/Kanban.jsx

```diff
@@ -90,23 +90,27 @@ export class Kanban extends React.Component {
     this.setState({ boardState: BOARD_STATE.LOADING })
     const { props } = this
 
-    const fetchRawFileContent = await handleFetchResult(
-      await getRawFileContent(
-        props.config.apiUrl,
-        props.content.workspace_id,
-        props.content.content_id,
-        props.content.current_revision_id,
-        props.content.label + KANBAN_FILE_EXTENSION
-      ),
-      true
-    )
-
-    if (fetchRawFileContent.apiResponse.ok && fetchRawFileContent.body.columns) {
-      this.setState({
-        boardState: BOARD_STATE.LOADED,
-        board: fetchRawFileContent.body || {}
-      })
-    } else {
+    try {
+      const fetchRawFileContent = await handleFetchResult(
+        await getRawFileContent(
+          props.config.apiUrl,
+          props.content.workspace_id,
+          props.content.content_id,
+          props.content.current_revision_id,
+          'kanban' + KANBAN_FILE_EXTENSION
```

Comment:
a const for `'kanban'`might be good, but not required

Reply:
Done.